src/lib/Commands/TestCommand/index.js

```diff
@@ -268,22 +285,32 @@ class TestCommand extends AbstractCommand {
     });
   }
 
-  /**
-   * Prepare source code
-   * @param testFile
-   * @return {{agent: string, device: string}}
-   * @private
-   */
+  // Prepare source code
+  // @param testFile
+  // @return {{agent: string, device: string}}
+  // @private
+
   _getSessionCode(testFile) {
     let agentCode, deviceCode;
 
-    /* [info] */
+    // [info]
     this._info(c.blue('Using ') + testFile.type + c.blue(' test file ') + testFile.name);
 
     // read/process test code
-    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim();
-    this._codeProcessor.variables.__FILE__ = path.basename(testFile.path);
-    testCode = this._codeProcessor.process(testCode);
+    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim().replace(LINE_AND_FILE_REGEXP, "@{__");
+    var environmentVars = "";
+    for (var prop in process.env) {
+      if (prop !== BUILD_API_KEY_ENV_VAR) { //deny to access for BUILD_API_KEY_ENV_VAR
+        let propValue = process.env[prop];
+        prop = prop.replace(UNSUPPORTED_SYMBOLS_REGEXP, 'X');
+        // Replace #{env:...} with @{...} if it is needed
+        let propertyRegExp = new RegExp('\\#\\{env:\\s*' + prop.replace(UNSUPPORTED_SYMBOLS_REGEXP, '.') + '\\s*\\}', 'g');
```

Comment:
You already replaced all these symbols to Xes on the line above. Here it makes no sense any more.